requirements/runtime.txt

```diff
@@ -1,3 +1,4 @@
+einops
```

Comment:
what's this for?

Reply:
When I deploy internlm-xcomposer2, it raise importError, asking `einops` to be installed.